src/buildShaderList.js

```diff
@@ -24,7 +24,12 @@ function listFilesSync(dir) {
 // return an object with the file path and file size
 function formatFileInfo(file) {
     const fileName = path.basename(file);
-    return fileName;
+    if (fs.existsSync(file + '.meta')) {
+        const meta = JSON.parse(fs.readFileSync(file + '.meta'));
+        return { "shaderName": fileName, "metaData": meta};
+    } else {
+        return { "shaderName": fileName};
```

Comment:
Minor thing, for readability, I'd use a leading space before closing the object;

```
        return { "shaderName": fileName, "metaData": meta };
```

```
return { "shaderName": fileName };
```